functional-tests/fn_bridge_deposit_happy.py

```diff
@@ -44,114 +127,46 @@ def main(self, ctx: flexitest.RunContext):
-        self.logger.debug(f"Deposit request tx: {tx}")
-        txid = btcrpc.sendrawtransaction(tx)
-        self.logger.debug(f"sent deposit request with txid = {txid} for address {el_address_1}")
-
-        # Now poll for the tx in chain
-        # TODO: replace by a wait_until btc rpc call
-        tx_published = False
-        for _ in range(10):
-            time.sleep(1)
-            try:
-                _ = btcrpc.gettransaction(txid)
-                print("Found expected tx in mempool")
-                tx_published = True
-                break
-            except Exception as e:
-                print(e)
-        assert tx_published, "Tx was not published"
-        # time to mature DRT
-        btcrpc.proxy.generatetoaddress(6, seq_addr)
-        time.sleep(1)
-        # TODO: time to mature DT
-        btcrpc.proxy.generatetoaddress(6, seq_addr)
-        time.sleep(1)
-        new_balance = int(rethrpc.eth_getBalance(f"0x{el_address_1}"), 16)
-        assert new_balance > original_balance_1, "balance did not increase"
-        # TODO: check that the difference is the same from the deposit amount.
-        #       You can get it from the rollups params
+        if new_address:
+            assert original_balance == 0, "balance is not zero"
+        else:
+            assert original_balance > 0, "balance is zero"
 
-        current_block_num = int(rethrpc.eth_blockNumber(), base=16)
-        print(f"Current reth block num: {current_block_num}")
+        # Generate Plenty of BTC to address
+        btcrpc.proxy.generatetoaddress(102, address)
 
-        # Second deposit, address 2
-        self.logger.debug(f"Deposit address: {addr_2}")
-        btcrpc.proxy.generatetoaddress(1, addr_2)
+        # Send DRT from Address 1 to EL Address 1
+        self.make_drt(ctx, el_address, bridge_pk)
+        # Make sure that the n_deposits is correct
 
-        # Create the first deposit request transaction
-        tx = deposit_request_transaction(el_address_2, btc_url, btc_user, btc_password)
-        self.logger.debug(f"Deposit request tx: {tx}")
-        txid = btcrpc.sendrawtransaction(tx)
-        self.logger.debug(f"sent deposit request with txid = {txid} for address {el_address_2}")
-
-        # Now poll for the tx in chain
-        tx_published = False
-        for _ in range(10):
-            time.sleep(1)
-            try:
-                _ = btcrpc.gettransaction(txid)
-                print("Found expected tx in mempool")
-                tx_published = True
-                break
-            except Exception as e:
-                print(e)
-        assert tx_published, "Tx was not published"
-        # time to mature FIXME: is this 6 blocks?
-        btcrpc.proxy.generatetoaddress(6, seq_addr)
-        time.sleep(1)
+        n_deposits_post = len(seqrpc.strata_getCurrentDeposits())
+        self.logger.debug(f"Current deposits: {n_deposits_post}")
+        assert n_deposits_post == n_deposits_pre + 1
```

Comment:
This assertion needs a failure message.

Reply:
Done in 1df4fb61dd69c50e6fdcfc65a281d1092000d54e